### PR TITLE
slack integration. closes #29

### LINKED ISSUE #29: Slack Integration

### PR DESCRIPTION
**Pull request by pwgraham91**
dcc21f7e5272c120c946d164316d374f0fb1a4b5ae66dfa32965c44aca18e5af

Slack integration which means that we will now be able to enter anyone's score. Because of this, I changed the way the website works to allow entering anyone's score again. People have complained about not being able to do this anymore and I think we can still encourage people to enter their own scores but it is not a requirement

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -54,8 +54,9 @@ def dict(self):
 class Game(db.Model):
     id = db.Column(db.Integer, primary_key=True)
 
-    created_at = db.Column(db.DateTime, nullable=False, default=sqlalchemy.func.now())
```
Comment (pwgraham91) [inline app/models.py:57]: this made migrations annoying. just entering it at time of game creation

File: app/views/handlers/game_handler.py [hunk 2]
```diff
@@ -1,25 +1,24 @@
+import datetime
 import trueskill
 
-from app.models import User, Game
+from app.models import Game
 
 
-def add_game(session, winner_id, loser_id, submitted_by_id):
-    if winner_id == loser_id:
+def add_game(session, winner, loser, submitted_by_id=None, slack_user_submitted_by=None):
```
Comment (zroyer) [inline app/views/handlers/game_handler.py:7]: can a game be added if submitted_by_id and slack_user_submitted_by are both None?

Comment (pwgraham91) [reply app/views/handlers/game_handler.py:7]: good catch. I changed it so it throws an exception below

**Comment by zroyer** [comment]
👍

**Pull request merged**